tests/python/frontend/onnx/test_forward.py

```diff
@@ -14,23 +14,35 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+# pylint: disable=import-self, invalid-name, unused-argument, unused-variable, missing-function-docstring, redefined-builtin, consider-using-f-string
```

Comment:
possible to not define some of these at file level? otherwise, reminder to add a comment here.